### ISSUE TITLE
Empty Blacklist: config.json created by setup command doesn't work

### ISSUE THREAD

**Issue by Inkrementator** [joeyates/imap-backup#143]
If I want to backup every folder, an empty blacklist achieves that. Doing that via setup subcommand creates an entry like this:

```
{
  "version": "2.0",
  "accounts": [
    {
      "username": "test@test.com",
      "password": "test",
      "local_path": "/test",
      "folders": [

      ],
      "folder_blacklist": true,
      "server": "test.com"
    }
  ]
}
```

That doesn't work. It will not download anything. Manually adding an empty object to the "folders" list fixes it.

```
{
  "version": "2.0",
  "accounts": [
    {
      "username": "test@test.com",
      "password": "test",
      "local_path": "/test",
      "folders": [
          {
          }

      ],
      "folder_blacklist": true,
      "server": "test.com"
    }
  ]
}
```

**Comment by joeyates** [comment]
Hi @Inkrementator thanks for opening the issue. I'll check what's happening.

**Comment by joeyates** [comment]
I've released version 9.0.2 to fix this.

When an account has `folder_blacklist` set but no list of configured folders, it now backs up all folders.
